homework_4/numpy_challenge.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+if __name__ == "__main__":
+   array1_tarrantion_movies = np.array(['Reservoir Dogs ', 'Pulp Fiction', 
+'Jackie Brown', 'Kill Bill: Vol. 1', 'Kill Bill: Vol. 2', 'Death Proof', 
+'Inglourious Basterds',  'Django Unchained', 'The Hateful Eight', 'Once 
+Upon a Time in Hollywoo'])
+   array2 = np.eye(5)
+   array3 = np.linspace(0, 25, 300)
+
+
+
+def matrix_multiplication(matrix1, matrix2):
+    try:
+        result = np.multiply(matrix1, matrix2)
+        return result
+    except ValueError:
+        return print('does not satisfy the condition of matrix 
+multiplication')
```

Comment:
1. Та же самая история, обычные строки просто так не переносятся
2. А зачем возвращать `None`?
```suggestion
        return print('does not satisfy the condition of matrix multiplication')
```